### PR TITLE
[MM-T2078]: add test: Profile picture: file types accepted

### PR DESCRIPTION
**Pull request by mr-orlik**
#### Summary
Profile picture: file types accepted: PNG, JPG, JPEG, BMP

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/18812

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: e2e/cypress/tests/integration/account_settings/profile/profile_picture_spec.ts [hunk 1]
```diff
@@ -89,4 +78,54 @@ describe('Profile > Profile Settings > Profile Picture', () => {
             should('be.visible').
             and('contain', 'Image limits check failed. Resolution is too high.');
     });
+
+    it('MM-T2078 Profile picture: file type PNG accepted', () => {
```
Comment (sinansonmez) [inline e2e/cypress/tests/integration/account_settings/profile/profile_picture_spec.ts:82]: I believe each of these tests below needs different test number (i.e.  MM-XXX). @saturninoabril please correct me if I am wrong here.

File: e2e/cypress/tests/integration/account_settings/profile/profile_picture_spec.ts [hunk 2]
```diff
@@ -16,20 +16,15 @@ import 'cypress-file-upload';
 import * as TIMEOUTS from '../../../fixtures/timeouts';
 
 describe('Profile > Profile Settings > Profile Picture', () => {
-    before(() => {
-        cy.apiInitSetup({loginAfter: true}).then(({offTopicUrl}) => {
+    beforeEach(() => {
+        cy.apiAdminLogin().apiInitSetup({loginAfter: true}).then(({offTopicUrl}) => {
```
Comment (sinansonmez) [inline e2e/cypress/tests/integration/account_settings/profile/profile_picture_spec.ts:20]: why do we need to login as admin?

**Review by sinansonmez — COMMENTED** [review]
Thanks for the great work. I just have two minor comments

**Comment by M-ZubairAhmed** [comment]
webapp is now part of mattermost monorepo now at https://github.com/mattermost/mattermost-server/tree/master/webapp Would you be interested in opening the PR over there?

**Comment by hmhealey** [comment]
The web app now lives in the `webapp/channels` over there. We decided to move it to be a monorepo to make both development and our CI pipelines easier, but it's unfortunately disruptive right now. We've updated [our setup docs](https://developers.mattermost.com/contribute/developer-setup/) and added [some steps for migrating to working with the monorepo](https://developers.mattermost.com/contribute/monorepo-migration-notes/).

Copying the changes over there hopefully shouldn't be too hard. There's a few changes to file paths (such as the code living in a dedicated `src` subdirectory), but most things should look the same as they do now. 

Let us know if you could use any help with this. Hopefully the process is smooth.

**Comment by M-ZubairAhmed** [comment]
Hello there! Could you please open the PR in the mattermost-server as part of the monorepo? I will be closing this PR for now. Thank you very much for your understanding and cooperation in this matter.

**Pull request closed without merging**